client/components/pack/AddItemModal.js

```diff
@@ -14,18 +14,6 @@ export const AddItemModal = ({
       trigger="Add Item"
       isActive={isAddItemModalOpen}
       onTrigger={setIsAddItemModalOpen}
-      footerButtons={[
```

Comment:
we should add the submit button from form to footer, will make UI cleaner

Reply:
most of the modal has like a button that makes api calls and also a button that closes modals, adding footer actually craates extra buttons for same functionality